moto/bedrock/models.py

```diff
@@ -0,0 +1,546 @@
+            "vpcConfig": self.vpc_config,
+        }
+        return {k: v for k, v in dct.items() if v}
+
+
+class CustomModel(BaseModel):
+    def __init__(
+        self,
+        model_name: str,
+        job_name: str,
+        job_arn: str,
+        base_model_arn: str,
+        hyper_parameters: Dict[str, str],
+        output_data_config: Dict[str, str],
+        training_data_config: Dict[str, str],
+        training_metrics: Dict[str, float],
+        base_model_name: str,
+        region_name: str,
+        account_id: str,
+        customization_type: Optional[str],
+        model_kms_key_arn: Optional[str],
+        validation_data_config: Optional[Dict[str, Any]],
+        validation_metrics: Optional[List[Dict[str, float]]],
+    ):
+        self.model_name = model_name
+        self.job_name = job_name
+        self.job_arn = job_arn
+        self.base_model_arn = base_model_arn
+        self.customization_type = customization_type
+        self.model_kms_key_arn = model_kms_key_arn
+        self.hyper_parameters = hyper_parameters
+        self.training_data_config = training_data_config
+        self.validation_data_config = validation_data_config
+        self.output_data_config = output_data_config
+        self.training_metrics = training_metrics
+        self.validation_metrics = validation_metrics
+        self.region_name = region_name
+        self.account_id = account_id
+        self.model_arn = f"arn:aws:bedrock:{self.region_name}:{self.account_id}:custom-model/{self.model_name}"
+        self.creation_time = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
+        self.base_model_name = base_model_name
+
+    def to_dict(self) -> Dict[str, Any]:
+        dct = {
+            "baseModelArn": self.base_model_arn,
+            "creationTime": self.creation_time,
+            "customizationType": self.customization_type,
+            "hyperParameters": self.hyper_parameters,
+            "jobArn": self.job_arn,
+            "jobName": self.job_name,
+            "modelArn": self.model_arn,
+            "modelKmsKeyArn": self.model_kms_key_arn,
+            "modelName": self.model_name,
+            "outputDataConfig": self.output_data_config,
+            "trainingDataConfig": self.training_data_config,
+            "trainingMetrics": self.training_metrics,
+            "validationDataConfig": self.validation_data_config,
+            "validationMetrics": self.validation_metrics,
+        }
+        return {k: v for k, v in dct.items() if v}
+
+
+class ModelInvocationLoggingConfiguration(BaseModel):
+    def __init__(self, logging_config: Dict[str, Any]) -> None:
+        self.logging_config = logging_config
+
+
+class BedrockBackend(BaseBackend):
+    """Implementation of Bedrock APIs."""
+
+    def __init__(self, region_name: str, account_id: str) -> None:
+        super().__init__(region_name, account_id)
+        self.ModelCustomizationJobs: Dict[str, ModelCustomizationJob] = {}
```

Comment:
Variables should all by `snake_case` across both Backends, to keep the style consistent